js/downloads.js

```diff
@@ -22,7 +22,7 @@ var packagesV8 = [hadoop2p7, hadoop2p6, hadoopFree, sources];
 // 2.4.0+
 var packagesV9 = [hadoop2p7, hadoop2p6, hadoopFree, scala2p12_hadoopFree, sources];
 
-addRelease("2.4.3", new Date("05/07/2019"), packagesV9, true);
+addRelease("2.4.4", new Date("08/30/2019"), packagesV9, true);
```

Comment:
This is yesterday which the vote passed.

Reply:
To reviewers, in the news, I'm using `2019-08-30` although today is 8/31. 
We can change it when this PR is decided to merge. *For now, SparkR is not published yet*.